.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,40 @@
+name: Feature Request
+description: Suggestions for new or different behavior.
```

i noticed that @gratcliff was creating tickets with this title, should we templatize this for features and bugs?

```suggestion
description: Suggestions for new or different behavior.
title: 'Feature Request: '
```

---

i'm cool with that change! honestly it feels a bit moot since we're automatically attaching labels and it's probably better to filter by those anyways

---

"Feature Request:" does add a ton of noise to bug ticket titles. what about `feat:` and `bug:` instead?

---

yes big fan of that change